app/controllers/api/v1/events_controller.rb

```diff
@@ -43,7 +43,8 @@ def destroy
   private
 
   def event_params
-    params.require(:event).permit(:name, :description, :address, :category_id, :user_id)
+		params.require(:event).permit(:name, :description, :address,
```

Comment:
I think you forgot to close the command

Reply:
Actually its due i put the params in different lines, since it was a bit long, below the comment you can see the rest

Reply:
Yeah, my bad. Watched it on the phone and it didn't appeared on next lines

Reply:
We're good then?